include/sdf/Mesh.hh

```diff
@@ -104,6 +108,20 @@ namespace sdf
     /// \param[in] _center True to center the submesh.
     public: void SetCenterSubmesh(const bool _center);
 
+    /// \brief Calculate and return the Inertial values for the Mesh
+    /// \param[in] density Density of the mesh in kg/m^3
+    /// \param[in] _autoInertiaParams ElementPtr to
+    /// <auto_inertia_params> element
+    /// \param[in] _config Parser Configuration
+    /// \param[out] _errors A vector of Errors object. Each object
+    /// would contain an error code and an error message.
+    /// \return A std::optional with gz::math::Inertiald object or std::nullopt
+    public: std::optional<gz::math::Inertiald>
+            CalculateInertial(double _density,
```

Comment:
Make `sdf::Errors` the first parameter

Reply:
Updated the API in all required places [4de7035](https://github.com/gazebosim/sdformat/pull/1304/commits/4de7035bab2d800d2ab6482c701cb2cac35778df)